src/transceivercontroller/TransceiverController.ts

```diff
@@ -78,7 +78,13 @@ export default interface TransceiverController {
   localVideoTransceiver(): RTCRtpTransceiver;
 
   /**
-   * Set [[RTCRtpEncodingParameters]] on the sender of transceiver
+   * Set [[RTCRtpEncodingParameters]] on the sender of transceiver.
+   * This method is called when a video is enabled/disabled or when there is the active speaker changes.
+   * The input parameters should not be empty.
+   * The encoding parameters for sender should be retrieved using sender.getParameters and updated using
+   * sender.setParameters method.
```

Comment:
It would be worth explicitly splitting these docs into:

* When this should be called by policies or controllers.
* How this is used by the default policy.
* What is the API contract (e.g., non-empty map).
* What implementations must do.